example/bottle/README.md

```diff
@@ -0,0 +1,33 @@
+# Bugsnag Bottle demo
```

Comment:
This new integration example does not seem to have any tests.

Reply:
This isn't a new integration as such - it uses wsgi - I just added an extra example whilst I was testing the runtime versioning.